docs/developer/README.md

```
@@ -117,48 +117,44 @@ You can deploy a DB container of different sizes using a DB_VERSION variable for
-- VMDISKSIZE: the disk size for the VM (default: 10G)
-- VMWAIT: the amount of minutes to wait on the console during first boot (default: 0)
+The `agent-vm` target accepts the following parameters:
 
-It is possible to create multiple VMs with different names:
+- `VMNAME`: the name of the VM to create (default: `flightctl-device-default`)
+- `VMRAM`: the amount of memory in MiB to allocate to the VM (default: `2048`)
+- `VMSSHPORT`: the host port forwarded to the VM's SSH (default: `2222`)
+
+It is possible to create multiple VMs with different names and ports:
 
 ```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)